### ISSUE TITLE
Exports failing 

### ISSUE THREAD

**Issue by justinholtweb** [solspace/craft3-freeform#768]
**Describe the bug or issue you're experiencing**
Exports of selected submissions fail with a SQL error:

```
`SQLSTATE[HY093]: Invalid parameter number: mixed named and positional parameters
The SQL being executed was: SELECT `s`.*, fc1.`first_name_1` as First_Name*, fc1.`last_name_2` as Last_Name*, fc1.`mailing_address_21` as Mailing_Address*, fc1.`city_9` as City*, fc1.`state_10` as State*, fc1.`zip_code_11` as Zip_Code*, fc1.`email_3` as Email*, fc1.`phone_20` as Phone*, fc1.`are_you_interested_in_financing_59` as Are_you_interested_in_financing?*, fc1.`would_you_like_a_sales_consultant_to_call_you_58` as Would_you_like_a_sales_consultant_to_call_you?*, fc1.`mbi_learnaboutmortonbuildings_47` AS `mbi_learnaboutmortonbuildings`, fc1.`building_category_2_22` as Building_Category_2*, fc1.`building_use_2_23` as Building_Use_2*, fc1.`build_location_zip_code_18` as Build_Location_Zip_Code*, fc1.`do_you_have_any_additional_comments_14` as Do_you_have_any_additional_comments?, fc1.`mbi_datecustomerwouldlikeprojectcompleted_49` AS `mbi_datecustomerwouldlikeprojectcompleted`, fc1.`mbi_buildingsitestatus_37` AS `mbi_buildingsitestatus`, fc1.`mbi_customerinvolvementintheproject_46` AS `mbi_customerinvolvementintheproject`, fc1.`mbi_interestedintouringmortonbuilding_45` AS `mbi_interestedintouringmortonbuilding`, fc1.`mbi_interestedinmeetingwithsc_44` AS `mbi_interestedinmeetingwithsc`, fc3.`gdpr_compliance_33` AS `gdprCompliance`, fc3.`token_24` AS `_token`, fc3.`form_action_32` AS `formAction`, fc3.`form_name_25` AS `formName`, fc3.`leadsourcecode_26` AS `leadsourcecode`, fc3.`new_secondaryleadsource_27` AS `new_secondaryleadsource`, fc3.`new_mediasource_28` AS `new_mediasource`, fc3.`date_lead_received_29` AS `Date_Lead_Received`, fc3.`refpage_30` AS `refpage`, fc3.`cd_visitorkey_31` AS `cd_visitorkey`, fc1.`form_return_url_35` AS `formReturnUrl`, fc3.`utm_source_55` AS `utm_source`, fc3.`utm_medium_54` AS `utm_medium`, fc3.`utm_content_53` AS `utm_content`, fc3.`utm_campaign_52` AS `utm_campaign`, fc2.`first_name_1` as First_Name*, fc2.`last_name_2` as Last_Name*, fc2.`mailing_address_21` as Mailing_Address*, fc2.`city_9` as City*, fc2.`state_10` as State*, fc2.`zip_code_11` as Zip_Code*, fc2.`email_3` as Email*, fc2.`phone_20` as Phone*, fc2.`build_location_zip_code_18` as Build_Location_Zip_Code*, fc2.`are_you_interested_in_financing_59` as Are_you_interested_in_financing?*, fc2.`would_you_like_a_sales_consultant_to_call_you_58` as Would_you_like_a_sales_consultant_to_call_you?*, fc2.`do_you_have_any_additional_comments_14` as Do_you_have_any_additional_comments?, fc2.`building_category_2_22` as Building_Category_2*, fc2.`building_use_2_23` as Building_Use_2*, fc2.`expected_timeline_to_start_19` as Expected_timeline_to_start*, fc3.`first_name_1` as First_Name*, fc3.`last_name_2` as Last_Name*, fc3.`mailing_address_21` as Mailing_Address*, fc3.`city_9` as City*, fc3.`state_10` as State*, fc3.`zip_code_11` as Zip_Code*, fc3.`email_3` as Email*, fc3.`phone_20` as Phone*, fc3.`are_you_interested_in_financing_59` as Are_you_interested_in_financing?*, fc3.`would_you_like_a_sales_consultant_to_call_you_58` as Would_you_like_a_sales_consultant_to_call_you?*, fc3.`building_category_2_22` as Building_Category_2*, fc3.`building_use_2_23` as Building_Use_2*, fc3.`build_location_zip_code_18` as Build_Location_Zip_Code*, fc3.`expected_timeline_to_start_19` as Expected_timeline_to_start*, fc3.`do_you_have_any_additional_comments_14` as Do_you_have_any_additional_comments?
FROM `freeform_submissions` `s`
LEFT JOIN `freeform_submissions_get_started_1` `fc1` ON fc1.`id` = s.`id`
LEFT JOIN `freeform_submissions_request_a_quote_2` `fc2` ON fc2.`id` = s.`id`
LEFT JOIN `freeform_submissions_storage_shed_3` `fc3` ON fc3.`id` = s.`id`
WHERE `s`.`id` IN ('2358111', '2358110', '2358109', '2358108', '2358107', '2358106', '2358105', '2358104', '2358103', '2358102', '2358101', '2358100', '2358099', '2358098', '2358097', '2358096', '2358050', '2358007', '2358006', '2358005', '2358004', '2357955', '2357904')
Error Info: Array
(
    [0] => HY093
    [1] => 0
)
↵
Caused by: PDOException
SQLSTATE[HY093]: Invalid parameter number: mixed named and positional parameters
in /var/www/html/vendor/yiisoft/yii2/db/Command.php at line 1302`
```

**Steps to reproduce**

1. Select some submissions on a form submissions listing.
2. Select "export as CSV"

**Expected behavior**
A CSV would be created with the specific selected entries. 

**Craft & Plugin Info (please complete the following information):**

- Craft Version: Craft Pro 4.4.15
- Freeform Version: 4.1.6
- Freeform Edition: Pro
- Fresh Install or Upgrade: Upgrade, didn't work in previous Craft 4 version installed. 

**Additional context**

**Comment by justinholtweb** [comment]
Any update on this? Client is getting itchy on exporting some data.

**Comment by kjmartens** [comment]
Hi @justinholtweb,

Sorry for the delay... 

I haven't seen an issue like this reported before. I have a few questions: 🙂 

1. Are you aware of any changes to the database you may have made?
2. Do you have any sort of custom module that interferes in some way?
3. Does this happen for every form or just the one?
4. Does using the Quick Export or Export Profiles feature work?
5. If either of those works, can you maybe send us an export of the data in a private support ticket for review?

**Comment by justinholtweb** [comment]
Thanks for the reply @kjmartens. Here is some background:
- We recently moved to Craft 4 for this client and also moved to a new server (Debian 10 on Cloudways using a Linode box). We are running MariaDB on the server. I can also reproduce the same behavior on my local install (DDEV w/ MariaDB)
- No manual changes to the database have been made. 
- This happens for every form (we have 6 forms)
- This happens with both the Quick Export and Export Profiles feature. I have tested with many and few selections of data, as well as a full export. Our particular use case will be for a fairly large export of 11k submissions (roughly)

I can send you a copy of the DB if it would help, or any other info to help debug the issue further.
